src/problem.rs

```diff
@@ -3,8 +3,8 @@ extern crate reqwest;
 
 use std::fmt::{Display, Formatter, Error};
 
-const PROBLEMS_URL: &str = "https://leetcode.com/api/problems/algorithms/";
-const GRAPHQL_URL: &str = "https://leetcode.com/graphql";
+const PROBLEMS_URL: &str = "https://leetcode-cn.com/api/problems/algorithms/";
+const GRAPHQL_URL: &str = "https://leetcode-cn.com/graphql";
```

Comment:
Can we add a command line argument to override the hostname?

Reply:
OK, I'll add it